diary/views.py

```diff
@@ -164,26 +164,74 @@ def get(self, request):
         request = requestSerial.validated_data
 
         # Diary 가져오기
-        diary_id = request.get('diaryId')
-        findDiary = Diary.objects.get(id=diary_id)
+        diaryId = request.get('diaryId')
+        findDiary = Diary.objects.get(id=diaryId)
 
         keywords = findDiary.keywords.all()
 
         # 모든 Sentence 와 연관된 Question 가져오기
-        question_keyword = []
+        result = []
 
         for keyword in keywords:
-            question_keyword.append({
-                "Q": keyword.questions.first().question,
-                "A": keyword.keyword
+            question = Questions.objects.get(keyword=keyword)
+            print(question)
+            result.append({
+                "questionId": question.pk,
```

Comment:
이거 print 지워주고
그 keyword 로 Question 직접 뽑지 말고
```suggestion
        for keyword in keywords:
            # 이렇게 된다면 이렇게하는게 직관적인듯
            question = keyword.questions.first()
            result.append({
                "questionId": question.pk,
```

Reply:
print 지우고 직관적으로 수정했습니다!